### DISCUSSION TITLE
forecasting: RMSes between series with different frequency

### DISCUSSION THREAD

**Discussion by nickcox896** [OTexts/Discussion#109] [General]
Hello,

I forecast inflation using monthly and quartely data and I obtain the correspondig RMSEs, using ARIMA models. 
 Are they comparable?

In both cases the dependent variable is the diifference of logarithm of the price index.

Many thanks in advance

**Comment by robjhyndman** [comment]
You can compare RMSEs on a test set across any models you like if they all have the same dependent variable. But comparing RMSEs on a training set is problematic if the models have different numbers of parameters.

**Reply by nickcox896** [reply]
OK, so the data frequency does matter as long as we have the same dependent variable. Thank you.
